lenient/include/module.h

```diff
@@ -96,6 +100,7 @@ module_validate_context_loaded(void)            \
     return 0;                                   \
 }
 
+// TODO to allow testing, consider if a signal can be fired instead of exiting
```

Comment:
Please add issue number of https://github.com/pottumuusi/c-encapsulation/issues/20 to this TODO comment.